src/sentry/utils/eventuser.py

```diff
@@ -125,6 +125,21 @@ def from_snuba(result: Mapping[str, Any]) -> EventUser:
             user_ident=result.get("user_id"),
         )
 
+    @classmethod
+    def for_tags(cls, project_id: int, values):
+        """
+        Finds matching EventUser objects from a list of tag values.
+
+        Return a dictionary of {tag_value: event_user}.
+        """
+        projects = Project.objects.filter(id=project_id)
+        result = {}
+        for value in values:
```

Comment:
Why aren't we hashing the tags like we do in the old function? https://github.com/getsentry/sentry/blob/fda93e57f0f2d311cfc7ecfda194236cf049a5f0/src/sentry/models/eventuser.py#L65

Reply:
Hmm, actually I don't think we need to hash... but we do probably just need to be looking at the `"sentry:user"` field, since that's the closest to the `hash` field in the `EventUser` model. It's set to `EventUser.tag_value` https://github.com/getsentry/sentry/blob/b8c0b8a0846b18a62d7b4c141ad2b3828218f67a/src/sentry/event_manager.py#L1034, which is the same value we hash into the `hash` column